### ISSUE TITLE
Add accessibilityIdentifier to all items

### ISSUE THREAD

**Issue by devinreams** [mozilla-lockbox/lockbox-ios#453]
To make XCUITests more resiliant to copy changes, and making L10n screenshots easier to write, each UIView that the user will interact with should have an `accessibilityIdentifier`.

To make a more accessible UI, each UIView that the user will interact with should have an `accessibilityLabel`.

**Comment by isabelrios** [comment]
@devinreams Do you agree on re-opening this issue? The accessibility Label part is fixed, almost all items have one, but not the accessibilityIdentifier. There are only a few. I added new ones in #303 but still there are more missing...and for some of them I would need devs help, not sure how to do it...

These are the only ones that we have so far(on that PR, not landed yet):
- Welcome screen:
getStarted.button

- Entry List
settings.button
sorting.button

-Settings List
sendUsageData.switch
lockNow.button

**Comment by devinreams** [comment]
>Do you agree on re-opening this issue? 

@isabelrios Yes! Sorry this was prematurely closed. I added a follow-on issue at #471
